readme.md

```diff
@@ -343,6 +343,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [Linux Containers](https://github.com/Friz-zy/awesome-linux-containers)
 - [Beginner-Friendly Projects](https://github.com/MunGell/awesome-for-beginners)
 - [Bluetooth Beacons](https://github.com/beaconinside/awesome-beacon)
+- [Spanish NLP](https://github.com/dav009/awesome-spanish-nlp)
```

Comment:
Hmm. I'm not sure. NLP is more common, to me, than LGTM. The only thing that might make it confusing is NeuroLinguistic Programming has the same acrostic. I can spell it out and add it, though.